src/maintainer/updating_pkgs.rst

```diff
@@ -32,7 +32,10 @@ When a new version of a package is released on PyPI/CRAN/.., we have a bot that
 #. Manually setting up git remotes:
 
    - Clone the conda-forge feedstock repository
-   - Add the remote of the bot: ``git add regro-cf-autotick-bot git://github.com/regro-cf-autotick-bot/<package>-feedstock.git``
+   - Add the remote of the bot: ``git add regro-cf-autotick-bot git@github.com:regro-cf-autotick-bot/<package>-feedstock.git``
+     .. important::
```

Comment:
This doesn't seem to be rendering properly on GH, does it need some whitespace above it?